### ISSUE TITLE
generate-ssh-hostkeys (and possibly others) not included in .deb or in setup.py install

### ISSUE THREAD

**Issue by andrewbogott** [andsens/bootstrap-vz#370]
```
Rebuilding initramfs
Installing startup scripts
[Errno 2] No such file or directory: '/usr/local/lib/python2.7/dist-packages/bootstrap_vz-0.9.10-py2.7.egg/bootstrapvz/common/assets/init.d/generate-ssh-hostkeys'

```

I only see this issue if I disable cloud-init, which may be a more-or-less unheard of use-case.  I suspect that this thread is the same issue:  https://lists.debian.org/debian-cloud/2016/06/msg00001.html

**Comment by andrewbogott** [comment]
Hm, this may be fixed in tip, I only see it in 0.9.9-1.

**Comment by andsens** [comment]
OK, closing...
